packages/common/models/dynamicLists-core.model.ts

```diff
@@ -20,16 +20,25 @@ import { BeagleDefaultComponent, Direction } from './types'
 
 export type NodeType = HTMLElement | null
 
+export interface TemplateItem {
+  case?: string,
```

Comment:
Here, we need to fully comply with the specification of the component ListView. It must be the same for all platforms.
```suggestion
  case?: string | boolean,
```

I understand it doesn't make sense to pass a boolean instead of an expression for this specific component. But, everytime an expression needs to be supported in the mobile platforms, it must be of type `Expression<type>` (iOS) or `Bindable<type>` (android), which means this can be either `type` or a string expression. It didn't make much sense to change this only for this component, so we must consider the weird scenario where the user would pass a boolean instead  of an expression.